### PR TITLE
Add Hands-on ML book

### PR DESCRIPTION
**Pull request by ageron**
I suggest adding my book to the list of TensorFlow books.  It covers everything from the Machine Learning basics using Scikit-Learn, to advanced Deep Learning topics (eg. Xavier/He initialization, batch normalization, data/model parallelism, RMS Prop, Adam optimization, the latest CNNs, RNNs, various Autoencoder architectures, Deep Q learning, and much more) using TensorFlow.  The code examples and exercise solutions are open sourced at https://github.com/ageron/handson-ml/.

### CODE REVIEW

**Comment by ageron** [comment]
Thanks @jtoy.
